### ISSUE TITLE
Twrp screenshot in wrong color

### ISSUE THREAD

**Issue by soilentgreen** [TeamWin/android_bootable_recovery#130]
- [ yes] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [ yes] I am running the latest version of TWRP
- [yes ] I have read the FAQ (https://twrp.me/FAQ/)
- [yes ] I have searched for my issue and it does not already exist

**Device codename**:  Samsung note 8 Greatlte 
**TWRP version**:  TWRP 3.5.1_9-0

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
 Make screenshot with power and volume down

#### WHAT IS THE EXPECTED RESULT?
Screenshot with "real" color

#### WHAT HAPPENS INSTEAD?
 All the red colors in the screenshot are blue

#### ADDITIONAL INFORMATION
I don't know why it's happening but recognized themer told me that when take a screenshot it inverts certain colors. Red becomes blue, the opposite on color chart.
It caused by a line in building for "non snapdragon" twrp.
For "normal" colors need to build on a lineage/ cyanogemmod , if its changed to qcomm the colors be "normal". But to do that need to have linux cmd as a first language.

**Comment by Dusk-Golem** [comment]
Hello, I have the same problem, I have galaxy A8 2018, running stock android 9, official twrp build 3.5.1_9.0. The screen shots are blue colored. See the attached pictures, the picture 6 is actually blue color but the screen shot makes it red, also the 6th one is in pink color...
![1](https://user-images.githubusercontent.com/63127881/111918121-6a255300-8aa5-11eb-830f-f621a760f7d0.png)
![2](https://user-images.githubusercontent.com/63127881/111918125-6eea0700-8aa5-11eb-82f4-d605bbe565ac.png)
![3](https://user-images.githubusercontent.com/63127881/111918128-71e4f780-8aa5-11eb-9869-1ca68e713e2f.png)
![4](https://user-images.githubusercontent.com/63127881/111918129-73aebb00-8aa5-11eb-8e86-a2618a5549c0.png)
![5](https://user-images.githubusercontent.com/63127881/111918340-7d84ee00-8aa6-11eb-89d5-fdce9122ec4c.png)
![6](https://user-images.githubusercontent.com/63127881/111918343-7eb61b00-8aa6-11eb-9589-c1b978add072.png)

**Comment by CaptainThrowback** [comment]
That doesn't look like standard TWRP. Are you using a theme?

**Comment by DarkKnight1995** [comment]
> That doesn't look like standard TWRP. Are you using a theme?

Yes I am using a theme. But look at the second last picture, it is the screenshot of the stock twrp theme (Blue Grey). Even stock twrp doesn't work correctly for screenshots...

**Comment by Dusk-Golem** [comment]
> That doesn't look like standard TWRP. Are you using a theme?

Yes I am using a custom theme pack. But look at the following screenshot which I just took after flashing standard twrp without any modification. The blue header color turned out to be yellow in the screenshot.
![Screenshot_2021-03-30-21-24-55 1](https://user-images.githubusercontent.com/63127881/113023174-d0456080-919e-11eb-992d-384c3546a6a1.png)

**Comment by Dusk-Golem** [comment]
isn't anyone working on this? This is really annoying when you want to share a custom theme screenshot with someone for help. PLEASE SOLVE THIS.

**Comment by CaptainThrowback** [comment]
> isn't anyone working on this? This is really annoying when you want to share a custom theme screenshot with someone for help. PLEASE SOLVE THIS.

This isn't a TWRP upstream issue. This is a device-specific issue that should be addressed with your maintainer and posted as an issue in your device's repo.

**Comment by Dusk-Golem** [comment]
> > isn't anyone working on this? This is really annoying when you want to share a custom theme screenshot with someone for help. PLEASE SOLVE THIS.
> 
> This isn't a TWRP upstream issue. This is a device-specific issue that should be addressed with your maintainer and posted as an issue in your device's repo.

mate, the OP had the problem on Galaxy Note 8 and my device is Galaxy A8 2018. Maybe this issue is samsung device specific? I don't know. Please see into this if possible. I have contacted the device maintainer for my device and he is willing to help but he is waiting for samsung to release latest kernel sources for updating device tree and such.
